db/migrate/20170728140134_remove_email_from_enterprises.rb

```diff
@@ -0,0 +1,21 @@
+class RemoveEmailFromEnterprises < ActiveRecord::Migration
+  def up
+    Enterprise.select([:id, :email]).each do |e|
```

Comment:
Migrations should not rely on models defined in the app since models change over time. If you want to use ActiveRecord features, you can define the model within the migration just with the methods you need.

Reply:
time to use https://github.com/testdouble/good-migrations @mkllnk ?

Reply:
Good idea @sauloperez. I don't like monkey patching, but it could help.